strings-csharp/HTMLContentString/HTMLContentString/HtmlAngle.cs

```diff
@@ -0,0 +1,31 @@
+using AngleSharp;
+using AngleSharp.Html.Parser;
+
+namespace HTMLContentString;
+
+public class HtmlAngle
+{
+    public async Task<string> GetHtmlAsStringAsync(string url)
+    {
+        var config = Configuration.Default.WithDefaultLoader();
+        var context = BrowsingContext.New(config);
+        var document = await context.OpenAsync(url);
+
+        return document.Body.InnerHtml;
+    }
+    public List<string> GetList(string htmlContent)
```

Comment:
@emekaemego 
There should be a new line before the next function (i.e the GetList() function)